spring-rabbit/src/main/java/org/springframework/amqp/rabbit/retry/RepublishMessageRecoverer.java

```diff
@@ -112,6 +124,16 @@ public void recover(Message message, Throwable cause) {
 		}
 	}
 
+	/**
+	 * Subclasses can override this message to add more headers to the republished message.
```

Comment:
s/message/method ?
Let me know and I'll fix on merge. This is only the concern to this PR.